CHANGELOG.md

```diff
@@ -4,6 +4,107 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## 0.13.0 - 2019-02-19
+
+### Added
+
+- When cluster-creation times out waiting for cluster-version
+  completion, the installer now logs the last failing-operator
+  message (if any).
+- The installer now invokes the [cluster-config
+  operator][cluster-config-operator] on the bootstrap machine to
+  generate `config.openshift.io` resources.
```

Comment:
only the custom resource definitions, the installer still creates the custom resources.

Reply:
> the custom resource definitions, the installer still creates the custom resources.

Fixed with b6882f2 -> 8f08508.